spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Viewing peeps' do
+  scenario 'Displaying all peeps' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    connection.exec("INSERT INTO peeps(post) VALUES('This is my first peep!');")
```

Comment:
I think it's best practice to swap 'connection.exec...' out for your 'Peep.add' class method